### PR TITLE
change machine-os-content periodics to use multi-stage steps

### PR DESCRIPTION
**Pull request by droslean**
Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml [hunk 1]
```diff
@@ -64,9 +64,7 @@ periodics:
         value: "4.1"
       - name: TAG_SOURCE
         value: machine-os-content:$(BRANCH)
-      - name: CLUSTER_TYPE
-        value: aws
-      - name: CONFIG_SPEC
+      - name: UNRESOLVED_CONFIG
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml:67]: You will also need to: 
1. remove the `--template` parameter
2. Remove the template volume mount
3. Remove the template volume

**Review by petr-muller — CHANGES_REQUESTED** [review]

**Comment by stevekuznetsov** [comment]
I would ask @AlexNPavel so that both of you are using the same automation to do this.

**Comment by petr-muller** [comment]
Eyeballed, looks good. I'm not 100% conviced that removing the config from the alt-arch jobs (that currently do not run ci-op at all) is a must, but it seems that the TODOs to make the jobs actually test somethign will not be fulfilled anytime soon, so /meh. I agree that Alex is the best person to review this.

**Review by AlexNPavel — APPROVED** [review]

**Pull request merged**